src/FSharpSource.targets

```diff
@@ -737,13 +737,24 @@
 
   <Choose>
     <When Condition="'$(BuildWith)' == 'LKG'">
-      <PropertyGroup>
-         <FSharpTargetsPath>..\lkg\FSharp-$(LkgVersion)\bin\Microsoft.FSharp.Targets</FSharpTargetsPath>
-         <FscToolPath>$(FSharpSourcesRoot)\..\lkg\bin\Debug\dnxcore50\win7-x64\publish</FscToolPath>
-         <FSCoreLKGPath>$(FSharpSourcesRoot)\..\lkg\FSharp-$(LkgVersion)\bin\FSharp.Core.dll</FSCoreLKGPath>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(TargetFramework)'=='portable47' OR '$(TargetFramework)'=='portable7' OR '$(TargetFramework)'=='portable78' OR '$(TargetFramework)'=='portable259' OR '$(TargetFramework)'=='coreclr'">
+		<PropertyGroup>
```

Comment:
Don, the idea is that we always use the coreclr version of the lkg for building the proto-compiler.  My next step, and I have a branch where this works except for the build task is to use the coreclr lkg to build a coreclr version of the proto compiler.  These are all steps towards making the whole repo ... including the windows bits, build cross platform.